src/App.tsx

```diff
@@ -1,10 +1,74 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+import { Notification } from './components/Notification/Notification';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+
+const USER_ID = 10140;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string | null>(null);
+
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
```

Comment:
I think we don't need this `useState`, we can create a simple `filteredTodos` variable that will be filled with filtered todos (this can be done in `useMemo`)

ask me in slack chat if you have some questions